WeScan/ImageScannerController.swift

```diff
@@ -71,6 +80,9 @@ public final class ImageScannerController: UINavigationController {
             navigationBar.tintColor = .black
         }
         navigationBar.isTranslucent = false
+        if (canSelect == true) {
```

Comment:
<!--
  0 failure: 
  1 warning:  `if`, `for`, `gua...
  
  
  DangerID: danger-id-Danger;
  File: WeScan/ImageScannerController.swift;
  Line: 83;
-->

- :warning: `if`, `for`, `guard`, `switch`, `while`, and `catch` statements shouldn't unnecessarily wrap their conditionals or arguments in parentheses. (`control_statement`)